Sources/PointFree/EnvVars.swift

```diff
@@ -1,8 +1,26 @@
 import Foundation
 
 enum EnvVars {
-  static let airbaseBase1 = ProcessInfo.processInfo.environment["AIRTABLE_BASE_1"] ?? "deadbeef-base-1"
-  static let airbaseBase2 = ProcessInfo.processInfo.environment["AIRTABLE_BASE_2"] ?? "deadbeef-base-2"
-  static let airbaseBase3 = ProcessInfo.processInfo.environment["AIRTABLE_BASE_3"] ?? "deadbeef-base-3"
-  static let airtableBearer = ProcessInfo.processInfo.environment["AIRTABLE_BEARER"] ?? "deadbeef-bearer"
+  private static var env: [String: String] {
+    return ProcessInfo.processInfo.environment
+  }
+
+  static let appSecret = env["APP_SECRET"] ?? "deadbeefdeadbeefdeadbeefdeadbeef"
+
+  enum BasicAuth {
+    static let username = env["BASIC_AUTH_USERNAME"] ?? "hello"
+    static let password = env["BASIC_AUTH_PASSWORD"] ?? "world"
+  }
+
+  enum Airtable {
+    static let base1 = env["AIRTABLE_BASE_1"] ?? "deadbeef-base-1"
+    static let base2 = env["AIRTABLE_BASE_2"] ?? "deadbeef-base-2"
+    static let base3 = env["AIRTABLE_BASE_3"] ?? "deadbeef-base-3"
+    static let bearer = env["AIRTABLE_BEARER"] ?? "deadbeef-bearer"
+  }
+
+  enum GitHub {
+    static let clientId = env["GITHUB_CLIENT_ID"] ?? "bf71e3ff1937fcd066d8"
+    static let clientSecret = env["GITHUB_CLIENT_SECRET"] ?? "94fb563ec0c21222db5d0254f228120030bf5bab"
```

Comment:
and our GitHub app!

now im sure it seems scary to have real id/secret in here, but this app just points back to `localhost:8080` so does that mean it's ok? if now I can delete this app and think of something else.

Reply:
I think it's prob ok?